spec/test_files/get_commit_sha_cfdd80f73d8c91121313d72227b4cbe283b57c1e.json

```diff
@@ -33,7 +33,17 @@
                 "type": "User",
                 "site_admin": false
             },
-            "labels": [],
+            "labels": [
```

Comment:
We now have a `next_release` label so I added it to this commit. We have another commit that doesn't have any labels and we are testing for it